src/main/java/com/team4/testingsystem/config/WebSecurityConfiguration.java

```diff
@@ -1,7 +1,7 @@
 package com.team4.testingsystem.config;
 
 import com.team4.testingsystem.security.JwtRequestFilter;
-import com.team4.testingsystem.services.CustomUserDetailsService;
+import com.team4.testingsystem.services.impl.CustomUserDetailsService;
```

Comment:
All the classes in services.impl package have Impl postfix. Let's use it here as well